Doc/c-api/tuple.rst

```diff
@@ -37,6 +37,17 @@ Tuple Objects
    or ``NULL`` with an exception set on failure.
 
 
+.. c:function:: PyObject* PyTuple_FromArray(PyObject *const *array, Py_ssize_t size)
+
+   Create a tuple of *size* items and copy references from *array* to the new
+   tuple.
+
+   * Return a new reference on success.
+   * Set an exception and return ``NULL`` on error.
```

Comment:
I think it does not need an item list. Just one or two plain sentences. Preferably in the style of `PyTuple_New()` and `PyTuple_Pack()`.

It should be explicitly documented that `array` can be NULL if `size` is 0.